CONTRIBUTING.md

```diff
@@ -71,12 +79,65 @@ The Login URI must point at `https://fxms-skylight.netlify.app/api/auth/login`,
 
 Documentation for Auth0 quickstart can be found at https://auth0.com/docs/quickstart/webapp/nextjs
 
-## Enabling Looker Locally
+## Enabling Looker locally
 
 Local development by default has Looker disabled, via the `IS_LOOKER_ENABLED` env var. It’s possible to enable it locally for testing, by setting `IS_LOOKER_ENABLED` to true && setting the correct Looker SDK credentials in `.env.local` (see `.env.sample` for the variables).
 
 Ask in #data-help on Slack to get Looker API credentials for your own Looker account to use in development.
 
+## Using browser tests to pull live message data into Skylight
+
+Skylight gets their live message data from asrouter in mozilla-central. For every new version in Release, we must be able to pull the live message data and commit it into Skylight by following these steps:
+
+1. Ensure that you have Firefox built by following the steps [here for macOS](https://firefox-source-docs.mozilla.org/setup/macos_build.html) or [here for Windows](https://firefox-source-docs.mozilla.org/setup/windows_build.html).
+2. In `mozilla-unified`, checkout to the release tag you want to pull data from.
+
+   To fetch all the tags, run:
+
+   ```
+   git cinnabar fetch --tags
+   ```
+
+   To see what specific tags exist, you can use regex and run a command similar to:
+
+   ```
+   git tag -l | grep 'FIREFOX_12[0-9]_0_RELEASE' | less
+   ```
+
+   Here is an example to checkout to a specific release:
+
+   ```
+   git checkout FIREFOX_129_0_RELEASE
+   ```
+
+3. [This patch](https://phabricator.services.mozilla.com/D201646) contains a browser test that will pull the JSON data for live messages from asrouter. Apply the patch to the release.
+
+   ```
+   moz-phab patch --apply-to here D201646
+   ```
+
+   Run `git log` to verify that the HEAD is pointing to the patch that's been applied.
+
+4. Clobber, build, and run the browser test using the following commands:
+
+   ```
+   ./mach clobber
+   ./mach build
+   ./mach test --headless browser/components/newtab/test/browser/browser_dump-provider-state.js
+   ```
+
+5. Assuming the test runs successfully, you will find the JSON output in `/tmp/file.json`. Commit the file into Skylight by moving the file to the appropriate location and renaming it in the format `{VERSION_NUM}-release.json`.
+
+   ```bash
+   # In Skylight
+   cd lib/asrouter-local-prod-messages/
+   mv /tmp/file.json ./{VERSION_NUM}-release.json
+   ```
+
+6. Run [`lib/mergeASRouterData.js`](/lib/mergeASRouterData.js) using node to merge all the release data into one file. Make sure you include the latest version number you've just collected data from into the `availableReleases` array inside the script.
```

Comment:
Please split this into two steps, with the first being to edit the script and add the version number, and the second step to run the script.